### DISCUSSION TITLE
Possibility of using treesitter-based textobjects without the syntax highlighting?

### DISCUSSION THREAD

**Discussion by scienceplease** [nvim-treesitter/nvim-treesitter-textobjects#379] [Q&A] [answered]
Is it possible to leverage `nvim-treesitter-textobjects` without forcing the syntax highlighting that comes from the parent `nvim-treesitter` plugin? The syntax highlighting `nvim-treesitter` performs can interfere with rendering in graphical UIs that are running Neovim embedded (e.g. [vscode-neovim](https://github.com/vscode-neovim/vscode-neovim)). ~~Unfortunately, setting `nvim-treesitter`'s `highlight = { enable = false }` completely disables `nvim-treesitter` along with this plugin as well.~~ Textobjects are incredibly useful even without syntax highlighting!

**Comment by kiyoon** [comment, marked as answer]
In my understanding setting highlight to false only disables the highlighting. I can tell that textobjects work for me without highlighting.

I think the documentation says it will disable the whole "highlight" extension of the nvim-treesitter

**Reply by scienceplease** [reply]
Sorry, you're absolutely right. I was running into an issue specific to `vscode-neovim` and mistakenly attributed it to `nvim-treesitter` as the README states "`false` will disable the whole extension". Thank you!
